roles/cifmw_snr_nhc/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+cifmw_snr_nhc_kubeconfig: "/home/zuul/.kube/config"
```

Comment:
nit: I know in many places we have /home/zuul, but we started to remove it or change to `/home/{{ ansible_user | default('zuul') }}/`